components/pickerbutton/stories/pickerbutton.stories.js

```diff
@@ -108,35 +111,75 @@ export default {
 		isQuiet: false,
 		isDisabled: false,
 		isFocused: false,
-		isKeyboardFocused: false,
 		iconType: "ui",
 		iconName: "ChevronDown",
-		position: "right"
+		position: "right",
 	},
 };
 
 export const Default = Template.bind({});
 Default.args = {};
 
 export const WithLabel = Template.bind({});
+WithLabel.storyName = "With label";
 WithLabel.args = {
 	label: "Select",
 };
 
 export const Disabled = Template.bind({});
-Disabled.tags = ["vrt-only"];
+Disabled.tags = ["!dev"];
 Disabled.args = {
-	isDisabled: true
+	isDisabled: true,
 };
 
 export const Quiet = Template.bind({});
-Quiet.tags = ["vrt-only"];
+Quiet.storyName = "Quiet with label";
+Quiet.tags = ["!dev"];
 Quiet.args = {
-	isQuiet: true
+	isQuiet: true,
+	label: "All",
```

Comment:
Adding the label here would change the chromatic snapshot, correct? It matches the docs site, so I like that update, but does that mean we should run chromatic on this branch? To update the baselines? And I guess if that's the case, modal might need to be checked as well (depending on what we decide to do with seeing the modal better).

Reply:
Correct. I have that noted as an expected VRT change, and VRT should be run to finalize the validation instructions. Thanks for calling that out! I've run the VRTs and will check on the differences.